resident/resident-service/src/main/java/io/mosip/resident/controller/IdentityController.java

```diff
@@ -94,7 +94,15 @@ public ResponseWrapper<Object> getInputAttributeValues(@PathVariable("schemaType
 		}
 		ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
 		String id = getIdFromUser();
-		Map<String, ?> propertiesResponse = identityUtil.getIdentityAttributes(id, schemaType, List.of());
+		Map<String, Object> propertiesResponse = identityUtil.getIdentityAttributes(id, schemaType, List.of());
+		identityUtil.getNameValueFromIdentityMapping()
```

Comment:
Can you not find the type to see if its an array or actual value?

Reply:
https://github.com/mosip/resident-services/blob/892fc80d32afa7d219fc0e878446db00ac96427c/resident/resident-service/src/main/java/io/mosip/resident/util/IdentityUtil.java#L246 
`public List<String> getNameValueFromIdentityMapping() `
method I have created to return string so if it contains only fullName the array will return only 1 value 
so added 
`if (identityUtil.getNameValueFromIdentityMapping().size() > SIZE) {`
it will not go into for each loop.